docs/podman-logs.1.md

```diff
@@ -15,7 +17,11 @@ any logs at the time you execute podman logs
 
 **--follow, -f**
 
-Follow log output.  Default is false
+Follow log output.  Default is false.
+
+Note: If you are following a container which is removed on exit
+`podman run --rm ...`, then there is a chance the the log file will be removed
+before `podman logs` reads the final content.
```

Comment:
This looks reasonable. I still find it confusing that removing a log file causes `tail` not to see final output -- my expectation is that `tail` keeps the file open, so unlink shouldn't matter -- but I'll chalk that up to Go weirdness. This note is helpful, thanks.

Reply:
If we reopen the file on an inotify event then that is a bug

Reply:
We aren't using inotify, even, just polling. There's a bug in the Go inotify library that causes it to eat delete events.